### PR TITLE
update links

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

File: pt/The-Manual/Tutorials-Examples/Simple-Acl-controlled-Application.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ Preparando sua Aplicação
 Primeiro, vamos pegar uma cópia recente do Cake.
 
 Para obter um download recente, visite o projeto CakePHP no Cakeforge:
-http://cakeforge.org/projects/cakephp/ e faça o download da versão
+https://cakeforge.org/projects/cakephp/ e faça o download da versão
```
Comment (othercorey) [inline pt/The-Manual/Tutorials-Examples/Simple-Acl-controlled-Application.rst]: Doesn't look like cakeforce.org supports https.  Also, these links seem to be invalid and open an online poker site now.

Should we remove them? @markstory

File: es/The-Manual/Developing-with-CakePHP/Installation.rst [hunk 2]
```diff
@@ -272,7 +272,7 @@ específico de un usuario o del *site*).
    En muchos servicios de hosting (GoDaddy, 1and1), tu servidor web está
    realmente siendo servido desde un directorio de usuario que ya
    utiliza mod\_rewrite. Si estás instalando CakePHP en un directorio de
-   usuario (http://ejemplo.com/~nombreusuario/cakephp), o en cualquier
+   usuario (https://ejemplo.com/~nombreusuario/cakephp), o en cualquier
```
Comment (othercorey) [inline es/The-Manual/Developing-with-CakePHP/Installation.rst]: These links appears to be invalid. Should we remove it? @markstory

File: es/The-Manual/Developing-with-CakePHP/Installation-Preparation.rst [hunk 3]
```diff
@@ -27,15 +27,15 @@ Todas las *releases* actuales están alojadas en CakeForge. Este *site*
...
+`https://cakeforge.org/projects/cakephp <https://cakeforge.org/projects/cakephp>`_.
 
 Se crean *nightly builds* alternativas que incluyen parches y mejoras al
 minuto (bueno, al día). Estas pueden ser accedidas desde la página
 principal de descargas aquí:
 `https://cakephp.org/downloads/index/nightly <https://cakephp.org/downloads/index/nightly>`_.
 Para actualizaciones realmente al minuto, puedes obtener el código
 directamente de la rama de desarrollo del repositorio git aquí:
-`http://code.cakephp.org/source <http://code.cakephp.org/source>`_.
+`https://code.cakephp.org/source <https://code.cakephp.org/source>`_.
```
Comment (othercorey) [inline es/The-Manual/Developing-with-CakePHP/Installation-Preparation.rst]: This link is invalid. I'm not sure where it came from @markstory

File: de/The-Manual/Beginning-With-CakePHP/Where-to-Get-Help.rst [hunk 4]
```diff
@@ -49,7 +49,7 @@ bring Deinen Sturzhelm mit.
 CakeForge
 =========
 
-`http://www.cakeforge.org <http://www.cakeforge.org>`_
+`https://www.cakeforge.org <https://www.cakeforge.org>`_
```
Comment (othercorey) [inline de/The-Manual/Beginning-With-CakePHP/Where-to-Get-Help.rst]: This appears to link to an online poker site now. Should we remove it? @markstory

File: pt/The-Manual/Tutorials-Examples/Simple-Acl-controlled-Application.rst (hunk 1 above)
Comment (markstory) [reply pt/The-Manual/Tutorials-Examples/Simple-Acl-controlled-Application.rst]: Yeah cakeforge has been dead for at least 8 years.

File: es/The-Manual/Developing-with-CakePHP/Installation.rst (hunk 2 above)
Comment (markstory) [reply es/The-Manual/Developing-with-CakePHP/Installation.rst]: Yeah we should remove them.

File: es/The-Manual/Developing-with-CakePHP/Installation-Preparation.rst (hunk 3 above)
Comment (markstory) [reply es/The-Manual/Developing-with-CakePHP/Installation-Preparation.rst]: code.cakephp.org predates our usage of github. There was a brief period in the projects history where we were hosted in a custom git server that one of the former contributors wrote.

File: de/The-Manual/Beginning-With-CakePHP/Where-to-Get-Help.rst (hunk 4 above)
Comment (markstory) [reply de/The-Manual/Beginning-With-CakePHP/Where-to-Get-Help.rst]: Yeah we don't need to send links to poker sites.

File: pt/The-Manual/Tutorials-Examples/Simple-Acl-controlled-Application.rst (hunk 1 above)
Comment (Arhell) [reply pt/The-Manual/Tutorials-Examples/Simple-Acl-controlled-Application.rst]: remove

File: es/The-Manual/Developing-with-CakePHP/Installation.rst (hunk 2 above)
Comment (Arhell) [reply es/The-Manual/Developing-with-CakePHP/Installation.rst]: remove

File: es/The-Manual/Developing-with-CakePHP/Installation-Preparation.rst (hunk 3 above)
Comment (Arhell) [reply es/The-Manual/Developing-with-CakePHP/Installation-Preparation.rst]: remove

File: de/The-Manual/Beginning-With-CakePHP/Where-to-Get-Help.rst (hunk 4 above)
Comment (Arhell) [reply de/The-Manual/Beginning-With-CakePHP/Where-to-Get-Help.rst]: remove

**Pull request merged**